### PR TITLE
Fix the styling of the frame in the Leaderboard section of Analytics.

### PR DESCRIPTION
**Pull request by AnnaMag**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Correct the styling in the dropdown menu item.

Closes #https://github.com/woocommerce/woocommerce/issues/32186 

### How to test the changes in this Pull Request:
1. Go to Analytics -> Leaderboards -> Ellipsis Menu -> Rows per table
2. Verify the layout:
Screenshot 2022-05-23 at 16 41 54

### CODE REVIEW

**Comment by botwoo** [comment]
:bar_chart: Test reports for this pull request have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33163/api/)
- E2E Test Report (TBC)

Latest commit referenced in the reports: `Add changelog` 165d3b659ab2136784e7f7c42285b79f2f022685
This comment will automatically be updated with the latest referenced commit when you push new changes to this pull request.

---

_Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them._

File: plugins/woocommerce-admin/client/dashboard/leaderboards/index.js
Comment (louwie17) [inline plugins/woocommerce-admin/client/dashboard/leaderboards/index.js:125]: Nice!! I thought this would require css changes, but adding a missing wrapper component is much nicer.

**Review by louwie17 — COMMENTED** [review]
Changes look great, the only think missing is a changelog (I am not sure how the changelog action actually passed).
You could run `pnpm nx changelog woocommerce-admin` to generate the changelog.
I tend to tag on the PR number at the end to in the description of the changelog, something like this:
> Fix the styling of the frame in the Leaderboard section of Analytics. #33163

Outside of that, this would be good to merge, I will approve after the changelog is merged. Great first PR @AnnaMag 🎉

File: plugins/woocommerce-admin/client/dashboard/leaderboards/index.js
Comment (AnnaMag) [reply plugins/woocommerce-admin/client/dashboard/leaderboards/index.js:125]: Thanks @louwie17! 
I added the changelog.

**Review by louwie17 — APPROVED** [review]
Great, thanks for pushing up the changelog. This is good to merge 🎉 🎆 🚢

**Pull request merged**
